### ISSUE TITLE
Audio player won't play after playing to the end.

### ISSUE THREAD

**Issue by towards-a-new-leftypol** [vector-im/element-web#25797]
### Steps to reproduce

1. Upload mp3
2. Press play, scrub or listen until the end
3. Try to play the track again, there is silence.

#### What did you expect?
I expect to be able to play an mp3 as many times as I like without refreshing the page.

#### What happened instead?
The audio gets to the end and the widget stops working, there is no more sound.

### Operating system

Linux

### Browser information

Firefox 115.0.1, behaviour also seen on my friends macbook with safari

### URL for webapp

Element version: 1.11.34 Olm version: 3.2.14

### Homeserver

Synapse 1.86.0

### Will you send logs?

No

**Comment by weeman1337** [comment]
@towards-a-new-leftypol  Unfortunately I cannot reproduce your issue. Please update your issue with more information. It would especially be useful to get the logs if you run into this issue. If we don’t hear back from you in a week, we will close the issue as we need to be able to reproduce it before we can fix it.

**Comment by weeman1337** [comment]
Tested with versions 1.11.34 and 1.11.36

**Comment by towards-a-new-leftypol** [comment]
I don't understand how to send debug logs. I have tried putting /rageshake into the room message input like the default issue description says and I get a popup that says "Unrecognized command" and asks me if I want to send it as a literal message.

I have also tried following the steps I found [here](https://ems-docs.element.io/books/element-cloud-documentation/page/submit-debug-logs) but I do not have a section to send debug logs in my Help & About settings.

I just checked this happens on chromium 114 as well.

**Comment by t3chguy** [comment]
Sounds like you don't have a rageshake server in your config.json - try reproduce on app.element.io then send debug logs.

**Comment by towards-a-new-leftypol** [comment]
Thanks @t3chguy I was able to reproduce on app.element.io and send the logs. Let me know if you can see them or if they are sufficient to debug the problem.

**Comment by weeman1337** [comment]
Thanks - that helped :+1: There is already another issue reporting the same. Will add the information over there.

**Comment by weeman1337** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/21359
